crates/cdk/src/mint.rs

```diff
@@ -762,3 +761,33 @@ impl From<MintKeySetInfo> for KeySetInfo {
         }
     }
 }
+
+fn create_new_keyset<C: secp256k1::Signing>(
+    secp: &secp256k1::Secp256k1<C>,
+    xpriv: ExtendedPrivKey,
+    derivation_path: DerivationPath,
+    unit: CurrencyUnit,
+    max_order: u8,
+) -> (MintKeySet, MintKeySetInfo) {
+    let keyset = MintKeySet::generate(
+        secp,
+        xpriv
+            .derive_priv(secp, &derivation_path)
+            .expect("RNG busted"),
+        unit,
+        max_order,
+    );
+    let keyset_info = MintKeySetInfo {
+        id: keyset.id,
+        unit: keyset.unit.clone(),
+        active: true,
+        valid_from: SystemTime::now()
```

Comment:
We can use the util function here 


https://github.com/cashubtc/cdk/blob/c8e61b34b8014946114823c435b8a086e559efb6/crates/cdk/src/util/mod.rs#L36-L41